pymongo/mongo_client.py

```diff
@@ -321,6 +321,14 @@ def __init__(
             Defaults to ``-1``, meaning no maximum. If maxStalenessSeconds
             is set, it must be a positive integer greater than or equal to
             90 seconds.
+          - `server_selector`: (callable) Optional, user-provided function that
+            augments server selection rules. The function should accept as an
+            argument a list of
+            :class:`~pymongo.server_description.ServerDescription` objects and
+            return a list of server descriptions that should be considered
+            suitable for the desired operation.
+
+          .. seealso:: :doc:`/examples/server_selection`
```

Comment:
server_selector is not really a Read Preference setting since it applies to both reads and writes. Should we move this to the "Other optional parameters can be passed as keyword arguments" section?

Reply:
Good point. Out of the current subsections we have in the `MongoClient` docstring, the 'Other optional parameters...` section seems most relevant for this option so I'll move it there.